roles/init_dbserver/tasks/init_dbserver.yml

```diff
@@ -1,15 +1,33 @@
 ---
 
-# setting variables available for all nodes
-# any of the variables existing can be updated and configured
+- name: Set the os variable
+  set_fact:
+    os: "{{ ansible_distribution | replace('Red Hat Enterprise Linux', 'RHEL') }}{{ ansible_distribution_major_version }}"
+
+- name: Reference pg_type variables
+  include_vars: "{{ pg_type }}.yml"
+
+- name: Reference ssl variables
+  include_vars: edb-ssl.yml
+  when: pg_ssl
+
+- name: Check support for Operating System
+  fail:
+    msg: "Operating System = {{ os }} not supported."
+  when: os not in supported_os
+
+- name: Check supported versions for Database engine
+  fail:
+    msg: "Database Engine Version = {{ pg_version }} not supported.
+          Supported versions are {{ supported_pg_version }}"
+  when: pg_version|int not in supported_pg_version
 
 - name: Set force_initdb based on the user input
```

Comment:
Shall we cleanup the force_initdb and create a new cleanup role?

Reply:
Same as https://github.com/EnterpriseDB/edb-ansible/pull/85#discussion_r546015786